### DISCUSSION TITLE
List speakers with incomplete info

### DISCUSSION THREAD

**Discussion by plaindocs** [pretalx/pretalx#1092] [Ideas] [answered]
Hi Folks,

I added CFP questions after the CFP closed, got get extra info from my speakers that I don't want everyone to have to fill out when submitting.

I'd like to have a selection in the speaker list that will show me speakers with incomplete info.

Cheers

**Comment by rixx** [comment, marked as answer]
New feature just dropped 👀
![2021-04-05T13:27+02:00](https://user-images.githubusercontent.com/2657547/113569393-bd091980-9612-11eb-8f60-abd5e6cc13c9.png)

**Comment by plaindocs** [comment]
That is pretty great, and speedy!

I'm guessing no link on my side is just an update issue? 

What I'd really like though, is a list of All speakers with ANY unanswered questions.

**Reply by rixx** [reply]
ANY unanswered questions is harder, particularly to translate into UI. I think generating email reminders for required questions is the closest we come currently, I'm afraid.

**Reply by plaindocs** [reply]
Seems a pretty great match for the drop-down in 
![image](https://user-images.githubusercontent.com/95874/113570894-8f719f80-9615-11eb-981a-b3e861064723.png)

![image](https://user-images.githubusercontent.com/95874/113570925-9d272500-9615-11eb-9dae-2da26d0ee688.png)

But I'll shut up if it is just me. 

Thanks again.

**Reply by rixx** [reply]
Questions are just one of the many pretalx features. Extending the drop-down by things like "speakers with no answered questions", "speakers without ALL answered questions" and so on would soon become unwieldy, because then you'd also get "speakers without avatar", "speakers without biography", "speakers with unconfirmed submissions" and so on. I really don't have a good solution :/
